server/index.js

```diff
@@ -1,27 +1,71 @@
+// const nr = require('newrelic');
 const express = require('express');
 const path = require('path');
-const bodyParser = require('body-parser');
+// const spdy = require('spdy');
+const fs = require('fs');
+const redis = require('redis');
+const redisClient = redis.createClient();
+const cassandra = require('cassandra-driver');
+const client = new cassandra.Client({ contactPoints: ['127.0.0.1'], localDataCenter: 'datacenter1', keyspace: 'restaurantmenu' });
+// const {
+//   getMenu,
+//   createMenu,
+//   updateMenu,
+//   deleteMenu,
+// } = require('./controllers');
 
 const app = express();
-const {
-  getMenu,
-  createMenu,
-  updateMenu,
-  deleteMenu,
-} = require('./controllers');
-
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
-app.use(express.static(path.join(__dirname, '../public')));
+// const options = {
+//   key: fs.readFileSync(__dirname + '/server.key'),
+//   cert: fs.readFileSync(__dirname + '/server.crt'),
+//   requestCert: false,
+//   rejectUnauthorized: false,
+// };
+const port = 3003;
+
+// spdy
+//   .createServer(options, app)
+//   .listen(port, () => {
+//     console.log('Listening on port: ' + port + '.');
+//   });
```

Comment:
lots of comments in this file. If they are still necessary, considering adding a comment explaining why they're here in the first place. If they're not necessary, remove them.